packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,344 @@
+/* Core root defaults */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+// Remove all animations, transitions and smooth scroll
+// for people that prefer not to see them
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+    scroll-behavior: auto;
+  }
+
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    scroll-behavior: auto !important;
+    transition-duration: 0.01ms !important;
+  }
+}
+
+/* Core body defaults */
+
+body {
+  background-color: var(--color-background);
+  color: var(--color-base);
+  font-family: var(--font-body);
+  font-size: var(--font-size-base);
+
+  // 👀 CMS-WDS/scout uses these reset styles
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  line-height: var(--font-line-height-base);
+  margin: 0;
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
```

Comment:
I can see the benefit in having a min-height of 100vh so you don't get weird footer placement